.travis.yml

```diff
@@ -77,6 +77,10 @@ before_install:
   - pushd builds
   # Build into own virtualenv
   # We therefore control our own environment, avoid travis' numpy
+  #
+  # Some change in virtualenv 14.0.5 caused `test_f2py` to fail. So, we have
+  # pinned `virtualenv` to the last known working version to avoid this failure.
```

Comment:
Would be good to figure out *why* it fails, but I don't have time today and
it doesn't sound like anyone else does, so oh well a temporary hack does
put off the problem :-)
.
(Adding an 'ls path/to/venv/bin' to a failing build would be informative
about what's going on...)
.
NB for clarity: "virtualenv" and "venv" are actually two different things
(venv's are virtualenv-like environments that are provided as a built-in
feature of recent py3 releases).


On Fri, Feb 5, 2016 at 12:42 PM, jakirkham <notifications@github.com> wrote:

> In .travis.yml
> <https://github.com/numpy/numpy/pull/7197#discussion_r52070007>:
>
> > @@ -77,6 +77,10 @@ before_install:
> >    - pushd builds
> >    # Build into own virtualenv
> >    # We therefore control our own environment, avoid travis' numpy
> > +  #
> > +  # Some change in virtualenv 14.0.5 caused `test_f2py` to fail. So, we have
> > +  # pinned `virtualenv` to the last known working version to avoid this failure.
>
> How does this comment look to you?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/numpy/numpy/pull/7197/files#r52070007>.
>



-- 
Nathaniel J. Smith -- https://vorpus.org <http://vorpus.org>